src/collections/map.rs

```diff
@@ -188,6 +213,30 @@ where
         Ok(())
     }
 }
```

Comment:
Just nitpicking but can we add an empty line here?

Reply:
I'm a bit concerned that fmt didn't catch this. This still passes fmt with the empty line added, but I do worry that down the road something like this will come up that will cause our CI to fail even though the clear cut answer to the cleaner solution causes a failure.

Reply:
I think fmt just doesn't have opinions about whether things should be separated by a line break, it just enforces not having 2 empty lines in a row.